modules/plugins/wfly-10/src/main/java/org/rhq/modules/plugins/wildfly10/BaseServerComponent.java

```diff
@@ -1135,4 +1138,23 @@ protected String collectPatches() {
             return ret.toString();
         }
     }
+
+    Integer getMinorVersion(){
+        String version = serverPluginConfig.getVersion();
```

Comment:
What's the point of this? We can already detect EAP 7.0 / EAP 7.1 difference by looking at the API version. 

https://github.com/rhq-project/rhq/blob/master/modules/plugins/wfly-10/src/main/java/org/rhq/modules/plugins/wildfly10/JBossProductType.java#L96

Lets not do multiple types of detection.